README.md

```diff
@@ -1,89 +0,0 @@
-# Sega Genesis Checksum Utility
```

Comment:
Why is this file deleted? Can we revert this?

Reply:
> Why is this file deleted? Can we revert this?
Yes, of course. I am adding it to my pull request.

Reply:
It looks like the pull request still tries to delete the README. Can you upload that one as well?

Once you do that and address the comments below, I'll be happy to merge this in.

Reply:
> It looks like the pull request still tries to delete the README. Can you upload that one as well?
> 
> Once you do that and address the comments below, I'll be happy to merge this in.

Did all of that.